### PR TITLE
P0 image input

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
Here I have added image and input elements.
this one contains a fairly less number of code changes

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Nice work, and refactor as well
Looks good to merge!

**Review by varnita619 — APPROVED** [review]
Good use of meta tags. Looks good to merge

File: atoms/colors/index.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import "/foundation/all.css";
```
Comment (SJTGSHIVAM) [inline atoms/colors/index.css]: can remove these imports for now I need to figure out better way this is creating problem. Need to look into it.

File: atoms/utils.css [hunk 2]
```diff
@@ -529,159 +530,159 @@
...
 }
 
 .tui__bw-t-xs {
-  border-top-width: var(--border-width-xs);
+  border-top-width: var(--g-border-width-xs);
   border-style: solid;
 }
 .tui__bw-t-sm {
-  border-top-width: var(--border-width-sm);
+  border-top-width: var(-sm);
```
Comment (SJTGSHIVAM) [inline atoms/utils.css]: fix needed

**Comment by SJTGSHIVAM** [comment]
As I have 2 positive reviews and also made the changes that were breaking. I'll be merging it now.

**Pull request merged**